src/User.php

```diff
@@ -44,6 +44,11 @@ public function getSanitizedBio(): string
         return str_replace("\n", '<br />', $bioStripped);
     }
 
+    public function getHashedPass(): string
+    {
+        return $this->hashed_pass;
+    }
+
```

Comment:
this method is no longer being used for anything, therefore it needs to be removed